vue/components/ui/listing-platforme/listing-platforme.vue

```diff
@@ -57,13 +57,13 @@
 }
 
 .scroll-button {
-    background: #ffffff url("~./assets/arrow-up.svg") no-repeat center;
+    background: $white url("~./assets/arrow-up.svg") no-repeat center;

```

Comment:
Can you please remove these color fixes? We'll fix it in every component in a different PR, as soon as the colors PR is merged.